content/en/includes/armory-operator/installation.md

```diff
@@ -0,0 +1,53 @@
+## Install the Armory Operator
+
+The Armory Operator has two distinct modes:
+
+- **Basic**: Installs Armory into a single namespace. This mode does not
+  perform pre-flight checks before applying a manifest.
+- **Cluster**: Installs Armory across namespaces with pre-flight checks to
+  prevent common misconfigurations. This mode requires a `ClusterRole`.
+
+Pick a release from
```

Comment:
```suggestion
Get the latest release from
```

Reply:
Received feedback from a client on not knowing which Operator releases are "deprecated" so may be better to have the user always get the latest release.